### ISSUE TITLE
missing translation General_Next in javascript of SitesManager

### ISSUE THREAD

**Issue by parruc** [piwik/piwik#8302]
I get these errors in SitesManager plugin:

<pre>
« The string General_Previous was not loaded in javascript. Make sure it is added in the Translate.getClientSideTranslationKeys hook.  0 - 10 of 5468 The string General_Next was not loaded in javascript. Make sure it is added in the Translate.getClientSideTranslationKeys hook. »
</pre>

**Comment by benjaminpick** [comment]
There is also a translation missing in the Menu under "Visitors": `Provider_SubmenuLocationsProvider`

**Comment by parruc** [comment]
Here everything seems translated in the Visitors submenu

**Comment by benjaminpick** [comment]
Maybe it is only missing in the german translation which I'm using?

**Comment by mattab** [comment]
@benjaminpick it's a different issue, see: https://github.com/piwik/piwik/issues/7985#issuecomment-105072277

**Comment by benjaminpick** [comment]
You're right, thank you.

**Comment by mattab** [comment]
Hi @parruc can you paste a screenshot showing the problem?

If you switch to english language do you still have the issue?

**Comment by parruc** [comment]
![screenshot from 2015-07-10 09 53 36](https://cloud.githubusercontent.com/assets/1440608/8614433/f73f933e-26e9-11e5-8959-d4da5fba369c.png)

I get this both in English and in Italian. With the proposed pull reqest I get the right translations.

**Comment by mattab** [comment]
I understand now, this string is loaded by "MultiSites" plugin and I guess you have disabled the plugin. so this fixes the bug, thanks

**Comment by mattab** [comment]
fixed in #8303

**Comment by parruc** [comment]
Yes I disabled it because atm I have 8k profiles and "all sites" view times out. ;-)

**Comment by mattab** [comment]
> Yes I disabled it because atm I have 8k profiles and "all sites" view times out. ;-)

@parruc we did a lot of work to improve this particular part of Piwik (make All Websites work better with thousands of websites). can you try:

1) Setup cron archiving (I guess you already did this): http://piwik.org/docs/setup-auto-archiving/ - which includes "disable browser trigger archiving"

2) re-enable MultiSites and try to load it in 2.14.0

How long does the page take to load for you?

Our goal is to make it usable even with 8k sites and it would be nice to have your return of experience how well this works for you

**Comment by parruc** [comment]
@mattab thanks for the suggestions:

1- We are aready using a conrjobbed archive and we have disabled browser trigger archiving.

2- I have tryied the 2.14 on the developement server and looks faster (it's slow only the first time and after it I suppose it's cached) I will let you know when we will update production. We can't do it right now because we have other activities to finish before.

3- We are planning to use piwik to monitor something like 50M pageviews per month. we are using (and works like a charm till now) a redis queue on the proxy that calls bulktracking. The slow part is the archiviation right now

4- A suggestion (but maybe it's better if I open a dedicated issue): With 8k users (one per prfoile) also the users view is terribily slow. Maybe an ajax pagination like the SitesManager.index view

**Comment by mattab** [comment]
> 4- A suggestion (but maybe it's better if I open a dedicated issue): With 8k users (one per prfoile) also the users view is terribily slow. Maybe an ajax pagination like the SitesManager.index view would solve the issue

See: #4474

For the rest, we look forward to hearing your feedback Re: Performance!

**Comment by parruc** [comment]
@mattab  I updated the productions server:
At the first run multisite plugin took something like 900 seconds to execute the query. Subsequent quesries (changing dates etc) were slightly faster but always over the minute. During these queries the  framed dashboard I'm using to present statistics was completly unresponsive so I had to disable again multisite.

These timings are referred to 9,5K profiles with a total of 2M pageview per month
